connector/src/main/java/org/geysermc/connector/entity/PlayerEntity.java

```diff
@@ -114,6 +118,35 @@ public void spawnEntity(GeyserSession session) {
         updateBedrockAttributes(session);
     }
 
+    public void sendAdventureSettings(GeyserSession session) {
```

Comment:
Since this only affects the Session player, I think it would make better sense in GeyserSession.